steps/cadence-innovus-signoff/scripts/generate-results.tcl

```diff
@@ -53,6 +53,11 @@ saveNetlist -excludeLeafCell                        \
 
 saveNetlist -excludeLeafCell $vars(results_dir)/$vars(design).vcs.v
 
+# Write netlist for Power-Ground Aware GL simulation
+
+saveNetlist -includePowerGround $vars(results_dir)/$vars(design).pg.v
+
```

Comment:
Note from the call: Make it $vars(design).vcs.pg.v

Reply:
Updated as discussed.